### DISCUSSION TITLE
Not able to run MFCMAPI session with Outlook 64 bit

### DISCUSSION THREAD

**Discussion by VladyslavAvramenko** [microsoft/mfcmapi#474] [General]
Not able to run MFCMAPI on Outlook 2016 64bit version. Tried both versions of MFCMAPI, the same error appears. Outlook is the default application, the quick repair was executed, tried online exchange mode, same results, new profiles (cached and online) - same issue.
Outlook 2016 > 12527.21594 > C2R
Office 365 ProPlus > Bitness = x64
Windows 10 Enterprise 64-bit > Ver.: 10.0.17134 

![mapierror2 (2)](https://user-images.githubusercontent.com/82175790/118791027-393b8180-b88e-11eb-9a61-d532459a165b.PNG)
![mapierror (3)](https://user-images.githubusercontent.com/82175790/118791032-39d41800-b88e-11eb-99b0-4b0059e25847.PNG)
![Teams_jCsSJBkvm4](https://user-images.githubusercontent.com/82175790/118791141-58d2aa00-b88e-11eb-9644-6e01acb3d447.png)

Thank you in advance

**Comment by stephenegriffin** [comment]
No point in testing with x86 since your Outlook is 64 bit. Does Office Repair fix things? If not, you might wanna get a procmon of the failure.

**Comment by VladyslavAvramenko** [comment]
Hello Stephen, 

I trust this email finds you well, and thank you very much for you reply. We have tried to perform quick repair, and issue still was exiting. Now I am testing online repair, and in case it will persist, I will proceed with O365 scruboff. I will share with you results, as soon I will have a feedback.

Wish you a lovely day.
With best regards, 
Vladyslav

email: ***@***.***


 

    среда, 19 мая 2021 г., 16:15:04 GMT+1, Stephen Griffin ***@***.***> написал(-а):  
 
 


No point in testing with x86 since your Outlook is 64 bit. Does Office Repair fix things? If not, you might wanna get a procmon of the failure.

**Comment by 12Knocksinna** [comment]
I'm also seeing errors after upgrading to version 2106 (Build 14131.20012).

![image](https://user-images.githubusercontent.com/45732130/121357827-0420cd00-c92a-11eb-90dd-dc52d4fccf4a.png)

Repairing didn't have any effect.

**Reply by 12Knocksinna** [reply]
A reboot after installing the latest version of Windows cured whatever problem existed.

**Comment by VladyslavAvramenko** [comment]
Hello everyone, 

In my case O365 reinstallation with scrubOff fixed the issue.
Thank you

Best regards,
Vlad

 

    четверг, 10 июня 2021 г., 13:58:13 GMT+1, Tony Redmond ***@***.***> написал(-а):  
 
 


A reboot after installing the latest version of Windows cured whatever problem existed.
